gems/pending/util/miq-process.rb

```diff
@@ -127,24 +127,7 @@ def self.processInfo(pid = nil)
   end
 
   def self.command_line(pid)
-    case Sys::Platform::IMPL
-    when :mswin, :mingw
```

Comment:
Yeah, it looks to use WMI, I didn't know if it returns the same thing though.  But, again, I don't know that anything is calling this on Windows.